packages/server/src/validator/utils.ts

```diff
@@ -14,7 +14,11 @@ export function validateParam(index: number | string, param: any, validation: an
     throw predefined.MISSING_REQUIRED_PARAMETER(index);
   }
 
-  if (param != null && Array.isArray(paramType)) {
+  if (param === null) {
+    throw predefined.INVALID_PARAMETER(index, `The value passed is not valid: ${param}.`);
```

Comment:
Hm, I think that according to the official schema, everything different than a string contating 20 byte hex should be rejected with this error. wdyt ?

Reply:
Do we want to throw this validation error even for parameters which are not required? It doesn't seem right.

If not, we can change `requiredIsMissing(param, validation.required)` which is already called above to also take `null` values into account:

```diff
export function requiredIsMissing(param: any, required: boolean) {
- return required && param === undefined;
+ return required && (param === undefined || param === null);
}
```

Reply:
> Hm, I think that according to the official schema, everything different than a string contating 20 byte hex should be rejected with this error. wdyt ?

@georgi-l95 do you mean in general? or for getcode in particular?

Reply:
@georgi-l95  This file is executed for every parameter, thats why I have added param == null here